src/main/java/org/sonarlint/intellij/issue/LiveIssue.java

```diff
@@ -24,6 +24,9 @@
 import com.intellij.openapi.util.TextRange;
 import com.intellij.psi.PsiFile;
 import java.security.MessageDigest;
+import java.util.Collections;
+import java.util.LinkedList;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.LinkedList'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)